README.md

```diff
@@ -68,24 +68,40 @@ struct MyMetric {
 
 As with any Telegraf point, tags are optional but at least one field is required.
 
+Timestamps are optional and can be set via the `timestamp` attribute:
+
+```rust
+use telegraf::*;
+
+#[derive(Metric)]
+struct MyMetric {
+    #[telegraf(timestamp)]
+    ts: u64,
+    field1: i32,
+}
+```
+
+If not present, the Telegraf daemon will set the timestamp using the current time.
+Timestamps are specified in nanosecond-precision Unix time, more information can be found [here](https://docs.influxdata.com/influxdb/v1.8/write_protocols/line_protocol_tutorial/#timestamp).
```

Comment:
mentioning that therefore they must be castable as a u64 might be nice as well now that I think about it